ToDoExercise/Content/app.js

```diff
@@ -0,0 +1,86 @@
+﻿
+var app = new Vue({
+
+    el: "#app",
+    components: {
+        litask,
+        modalcomponent,
+        draggable: window['vuedraggable']      //esto se hace ya que vuedraggable lo utilizamos a través de cdn, con lo que no había otra forma de utilizarlo...
+    },
+    data() {
+        return {
+            modalInput: {},
+            forms: [],
+            counter: 0
+        }
+    },
+    methods: {
+
+        createCard() {
+            this.incrementCounter()
+            this.forms.push(this.createEmptyCardObject())
+            this.saveLocal();
+        },
+        deleteTaskFromId(id) {
+            this.forms.splice(this.getIndexFromId(id), 1)
+            this.saveLocal();
+        },
+        updateTaskFromObject(modal) {
+            this.forms.splice(this.getIndexFromId(modal.id), 1, this.editFormById(modal.title, modal.description, modal.name, modal.id))
```

Comment:
no debería hacer falta usar editFormById
this.tasks.splice(this.tasks.findIndex(task.id), 1, task)